cleanrl/sac_atari.py

```diff
@@ -0,0 +1,359 @@
+        help="weather to capture videos of the agent performances (check out `videos` folder)")
+
+    # Algorithm specific arguments
+    parser.add_argument("--env-id", type=str, default="Seaquest-v4",
+        help="the id of the environment")
+    parser.add_argument("--total-timesteps", type=int, default=120000,
+        help="total timesteps of the experiments")
+    parser.add_argument("--buffer-size", type=int, default=int(1.5e5),
+        help="the replay memory buffer size") # smaller than in original paper but evaluation is done only for 100k steps anyway
+    parser.add_argument("--gamma", type=float, default=0.99,
+        help="the discount factor gamma")
+    parser.add_argument("--tau", type=float, default=1.0,
+        help="target smoothing coefficient (default: 1)") # Default is 1 to perform replacement update
+    parser.add_argument("--batch-size", type=int, default=64,
+        help="the batch size of sample from the reply memory")
+    parser.add_argument("--learning-starts", type=int, default=2e4,
+        help="timestep to start learning")
+    parser.add_argument("--policy-lr", type=float, default=3e-4,
+        help="the learning rate of the policy network optimizer")
+    parser.add_argument("--q-lr", type=float, default=3e-4,
+        help="the learning rate of the Q network network optimizer")
+    parser.add_argument("--policy-frequency", type=int, default=4,
+        help="the frequency of training policy (delayed)")
+    parser.add_argument("--target-network-frequency", type=int, default=8000, # Denis Yarats' implementation delays this by 2.
+        help="the frequency of updates for the target networks")
+    parser.add_argument("--alpha", type=float, default=0.2,
+        help="Entropy regularization coefficient.")
+    parser.add_argument("--autotune", type=lambda x:bool(strtobool(x)), default=True, nargs="?", const=True,
+        help="automatic tuning of the entropy coefficient")
+    args = parser.parse_args()
+    # fmt: on
+    return args
+
+
+def make_env(env_id, seed, idx, capture_video, run_name):
+    def thunk():
+        env = gym.make(env_id)
+        env = gym.wrappers.RecordEpisodeStatistics(env)
+        if capture_video:
+            if idx == 0:
+                env = gym.wrappers.RecordVideo(env, f"videos/{run_name}")
+        env = NoopResetEnv(env, noop_max=30)
+        env = MaxAndSkipEnv(env, skip=4)
+        env = EpisodicLifeEnv(env)
+        if "FIRE" in env.unwrapped.get_action_meanings():
+            env = FireResetEnv(env)
+        env = ClipRewardEnv(env)
+        env = gym.wrappers.ResizeObservation(env, (84, 84))
+        env = gym.wrappers.GrayScaleObservation(env)
+        env = gym.wrappers.FrameStack(env, 4)
+        env.seed(seed)
+        env.action_space.seed(seed)
+        env.observation_space.seed(seed)
+        return env
+
+    return thunk
+
+
+# He initialization
+def network_init(net, bias_const=0.0):
```

Comment:
Just nitpicking, but could we use `layer_init` instead, as in the other scripts:
https://github.com/vwxyzjn/cleanrl/blob/e466f6efb251462de2b80f064133f32cb5d83e22/cleanrl/ppo_continuous_action.py#L102

Thanks.

Reply:
I'm definitely going to change to a `layer_init`-style function.
The paper does explicitly mention kaiming initialization, so I'll leave that. Is that okay with you?